index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Video Streaming</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+</head>
+<body>
+    <nav class="navbar" id="navbar">
+        <ul class="left">
+            <li><i class="fas fa-bars icon"></i></li>
+            <li><i class="fab fa-youtube youtube-icon" ></i>Youtube</li>
+        </ul>
+       <div class="search">
+           <input type="text" name="" id="" placeholder="Search for Video">
+           <button class="btn"><i class="fas fa-search"></i></button>
+       </div>
+
+       <ul class="right">
+           <li><i class="fas fa-video"></i></li>
+           <li><i class="fas fa-th"></i></li>
+           <li><i class="fas fa-comment"></i></li>
+           <li><i class="fas fa-bell"></i></li>
+       </ul>
+    </nav>
+    <main class="main-section">
+        <div class="media">
+            <div class="video" style="position:relative;padding-top:56.25%;">
+                <iframe width="100%" height="100%" src="https://www.youtube.com/embed/ZnuwB35GYMY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="position:absolute;top:0;left:0;width:100%;height:100%;"></iframe>
```

Comment:
Width and height should be in pixels,frameborder is obsolute.
Again no inline styling please. Position absolute isn't necessary, you can use a different way to center your content.

Reply:
Thank a lot for the review, we're going to make the changes